extensions/file/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,4 +0,0 @@
-module.name=file
```

Comment:
I would introduce only the changes for the new test runner and change tests in a separated PR once the rest of the team is aware of how it works